templates/prometheus.yaml

```diff
@@ -35,6 +35,19 @@ spec:
                 port: metrics
                 interval: 30s
         prometheusSpec:
+          additionalScrapeConfigs:
+            - job_name: 'kubernetes-nodes-containerd'
+              metrics_path: v1/metrics
```

Comment:
can you do `/v1/metrics` here. Otheriwse, the target in prometheus shows a bit weird: `https://<ip>:1338v1/metrics`, though it works.

Reply:
Good 👁 ! Done